packages/edit-navigation/src/store/actions.js

```diff
@@ -184,7 +186,7 @@ function* batchSave( menuId, menuItemsByClientId, navigationBlock ) {
 	);
 
 	return yield apiFetch( {
-		url: '/wp-admin/admin-ajax.php',
+		url: ajaxurl,
```

Comment:
```suggestion
		url: ajaxurl || '/wp-admin/admin-ajax.php',
```

As this relies on the window (easily manipulated) should we include a fallback?

Reply:
Thanks, @getdave.

I've added a fallback.